pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -730,6 +730,17 @@ func (h *Helper) updateConfig(configDir string, opt *StartOptions) error {
 		nodeCfg.DNSIP = ""
 	}
 	nodeCfg.DNSBindAddress = ""
+
+	// Set the cgroup driver from the current docker
+	cgroupDriver, err := h.dockerHelper.CgroupDriver()
+	if err != nil {
+		return err
+	}
+	if nodeCfg.KubeletArguments == nil {
+		nodeCfg.KubeletArguments = configapi.ExtendedArguments{}
+	}
+	nodeCfg.KubeletArguments["cgroup-driver"] = []string{cgroupDriver}
```

Comment:
@stts this is just setting it to whatever docker reports is its cgroupDriver instead of the hardcoded systemd value. (only for the case of cluster up)